test/test_maxwellDriver1d.py

```diff
@@ -1057,3 +1058,64 @@ def test_buildDrivedEvolutionOperator():
     plt.spy(A_drived, markersize=4)
     # plt.spy(A, markersize=4)
     plt.show()
+
+def test_check_initial_conditions_GW_right():
+
+    x_min = -4.0
+    x_max = 4.0
+    k_elements = 400
+
+    sp = FDTD1D(mesh=Mesh1D(x_min, x_max, k_elements, boundary_label="PEC"))
+    driver = MaxwellDriver(sp, timeIntegratorType='LF2', CFL=1.0)
+
+    s0 = 0.25
+    initialFieldE = np.exp(-(sp.x)**2/(2*s0**2))
+    initialFieldH = np.exp(-(sp.xH)**2/(2*s0**2))
+
+    d = np.diff(sp.x)[0]
+    n_sigmas=3
+    
+    driver['E'][:] = initialFieldE[:]
+    driver['H'][:] = initialFieldH[:]
+
+    x_center = k_elements / 2
+    one_time_unit_elements = k_elements / (x_max - x_min)
+
+    E_gaussian_start = driver['E'][int(x_center - n_sigmas*s0 / d)\
+                                          :int(x_center + n_sigmas*s0 / d)]
+    
+    H_gaussian_start = driver['H'][int(x_center-1 - n_sigmas*s0 / d)\
+                                          :int(x_center-1 + n_sigmas*s0 / d)]
+
+    driver.run_until(1.0)
+
+    E_gaussian_end = driver['E'][int(x_center+one_time_unit_elements - n_sigmas*s0 / d)\
+                                          :int(x_center+one_time_unit_elements + n_sigmas*s0 / d)]
+    
+    H_gaussian_end = driver['H'][int(x_center+one_time_unit_elements-1 - n_sigmas*s0 / d)\
+                                          :int(x_center+one_time_unit_elements-1 + n_sigmas*s0 / d)]
+
+    mask= int(2*n_sigmas*s0 / d)
+    tolerance = 0.5
```

Comment:
This tolerance is really low... This is not enough to pass a test. They wave after traveling has to  be similar to the original not "half" of it.